SKILL.tsv

```diff
@@ -5237,7 +5237,7 @@ SKILL_20150918_005236	[Paint] Increases the chance what enemies turn around by 1
 SKILL_20150918_005237	$Deals 10% additional damage to Plant-type monsters when equipped with a [Blunt] type weapon.
 SKILL_20150918_005238	Inflict damage of 50% per the physical damage when knocking flying enemies out of the air, if attacked by [Smite] skill. (The effect only activates when the attribute is on)
 SKILL_20150918_005239	Conversion: Might Enhance
-SKILL_20150918_005240	Physical attack of monsters coverted with [Coversion] increase by 50% per the strength of character.
+SKILL_20150918_005240	Physical attack of monsters converted with [Conversion] increases by 50% of the strength of character.
```

Comment:
Increases 50% of the monster's physical attack converted with [Conversion] equal to 50% of the character's STR.